.github/workflows/upload-models.yaml

```
@@ -1234,6 +1234,35 @@ jobs:
 
           mv models/* .
 
+      - name: NeMo CTC Persian (shenava)
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)